docs/FAQ.md

```diff
@@ -27,28 +27,28 @@ these changes and release updates as quickly as possible.
 ### Which distributions are supported?
 
 The RAPIDS Accelerator for Apache Spark officially supports
-[Apache Spark](get-started/getting-started.md),
-[Databricks Runtime 7.0](get-started/getting-started-with-rapids-accelerator-on-databricks.md)
-and [Google Cloud Dataproc](get-started/getting-started-gcp.md).
+[Apache Spark](get-started/getting-started-on-prem),
+[Databricks Runtime 7.0](get-started/getting-started-databricks)
+and [Google Cloud Dataproc](get-started/getting-started-gcp).
```

Comment:
This broke the links.  For example, visit https://github.com/sameerz/spark-rapids/blob/branch-0.2-documentation-updates/docs/FAQ.md and try to click on any of these links.  Tack `.md` on the end of any of the URLs that aren't found and it finds them.

This comment applies to all the places where `.md` was removed.

Reply:
Added the .md links back.  Testing locally with Jekyll they do not work.  I will have to test again with gh-pages.